EIPs/eip-4.md

```diff
@@ -2,52 +2,52 @@
   EIP: 4
   Layer: Process
   Title: EIP Classification
-  Author: Eric Lombrozo
+  Author: Joseph Chow
   Status: Draft
   Type: Process
   Created: 2015-11-17
 </pre>
 
-==Abstract==
+# Abstract
 
-This document describes a classification scheme for EIPs, adapted from BIP123.
+This document describes a classification scheme for EIPs, adapted from BIP123 by Eric Lombrozo.
```

Comment:
Doesn't this now read that BIP123 was adapted to EIP4 by Eric Lombrozo?

Reply:
yes, will just reference with an inline link, thanks.